instrumentation/http-url-connection/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/httpurlconnection/GetOutputStreamContext.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.httpurlconnection;
+
+import static io.opentelemetry.context.ContextKey.named;
+
+import io.opentelemetry.context.Context;
+import io.opentelemetry.context.ContextKey;
+import io.opentelemetry.context.ImplicitContextKeyed;
+import java.net.HttpURLConnection;
+
+public class GetOutputStreamContext implements ImplicitContextKeyed {
+  private static final ContextKey<GetOutputStreamContext> KEY =
+      named("opentelemetry-http-url-connection-get-output-stream");
+
+  private boolean outputStreamMethodOfSunConnectionCalled;
```

Comment:
probably should be volatile?

Reply:
Added, I did not notice that `GetOutputStreamContext `could be accessed by severral threads at the same time